doc/nrf/releases_and_maturity/releases/release-notes-2.7.0.rst

```diff
@@ -0,0 +1,1238 @@
+
+  * Added support for the :ref:`zephyr:nrf54l15pdk_nrf54l15` board.
+  * Changed the default :ref:`feature set <thread_ug_feature_sets>` from Master to RCP.
+
+Zigbee samples
+--------------
+
+* Updated the Zigbee samples that used :ref:`zephyr:nrf-ieee802154-rpmsg-sample` or :ref:`multiprotocol-rpmsg-sample` radio core firmware, so that they now use the :ref:`ipc_radio`.
+
+Wi-Fi samples
+-------------
+
+* Added:
+
+  * The :ref:`softap_wifi_provision_sample` sample.
+  * The :ref:`wifi_thread_coex_sample` sample that demonstrates Wi-Fi and Thread coexistence.
+  * The :ref:`wifi_promiscuous_sample` sample that demonstrates how to set Promiscuous mode, establish a connection to an Access Point (AP), analyze incoming Wi-Fi packets, and print packet statistics.
+
+* Updated the Wi-Fi samples that used :ref:`zephyr:bluetooth-hci-ipc-sample` or :ref:`zephyr:nrf-ieee802154-rpmsg-sample` radio core firmware, so that they now use the :ref:`ipc_radio`.
+
+* :ref:`wifi_shell_sample` sample:
+
+  * Added:
+
+    * ``Auto-Security-Personal`` mode to the ``connect`` command.
+    * Support for the ``WPA-PSK`` security mode to the ``wifi_mgmt_ext`` library.
+
+  * Updated the ``connect`` command to provide better control over connection parameters.
+
+* :ref:`wifi_station_sample` sample:
+
+  * Modified to use the :ref:`lib_wifi_ready` library to manage the Wi-Fi use.
+
+Other samples
+-------------
+
+* Added the :ref:`coremark_sample` sample that demonstrates how to easily measure a performance of the supported SoCs by running the Embedded Microprocessor Benchmark Consortium (EEMBC) CoreMark benchmark.
+
+* :ref:`bootloader` sample:
+
+  * Added support for the :ref:`zephyr:nrf9151dk_nrf9151` and the :ref:`zephyr:nrf9161dk_nrf9161` boards.
+
+* :ref:`ipc_service_sample` sample:
+
+  * Removed support for the `OpenAMP`_ library backend on the :ref:`zephyr:nrf54h20dk_nrf54h20` board.
+
+Drivers
+=======
+
+This section provides detailed lists of changes by :ref:`driver <drivers>`.
+
+Wi-Fi drivers
+-------------
+
+* Added support for random MAC address generation at boot using the :kconfig:option:`CONFIG_WIFI_RANDOM_MAC_ADDRESS` Kconfig option.
+* Removed support for setting RTS threshold through the ``wifi_util`` command.
+
+Libraries
+=========
+
+This section provides detailed lists of changes by :ref:`library <libraries>`.
+
+Binary libraries
+----------------
+
+.. _lib_bt_ll_acs_nrf53_readme:
+
+* Removed the LE Audio controller for nRF5340 library, which was deprecated in the :ref:`v2.6.0 release <ncs_release_notes_260>`.
+  As mentioned in the :ref:`migration_2.6`, make sure to transition to Nordic Semiconductor's standard :ref:`ug_ble_controller_softdevice` (:ref:`softdevice_controller_iso`).
+
+Bluetooth libraries and services
+--------------------------------
+
+* :ref:`bt_mesh` library:
+
+  * Updated the :ref:`bt_mesh_light_ctrl_srv_readme` model documentation to explicitly mention the Occupany On event.
```

Comment:
```suggestion
  * Updated the :ref:`bt_mesh_light_ctrl_srv_readme` model documentation to explicitly mention the Occupancy On event.
```

Reply:
```suggestion
  * Updated:

    * The :ref:`bt_mesh_light_ctrl_srv_readme` model documentation to explicitly mention the Occupancy On event.
    * The :ref:`bt_mesh_light_ctrl_srv_readme` to instantly change the current or the target lightness level if the state machine is in or transitioning to one of the Run, Prolong or Standby states when receiving a Light LC Property Set message (as a result of calling the function :c:func:`bt_mesh_light_ctrl_cli_prop_set`) modifying the relevant state's level.
```
Based on changes from this PR: https://github.com/nrfconnect/sdk-nrf/pull/16047